Tests/Realm.Tests/Sync/QueryBasedSyncTests.cs

```diff
@@ -455,12 +454,55 @@ public void WaitForSynchronization_OnBackgroundThread_Throws(bool openAsync)
             });
         }
 
+        [TestCase(true)]
+        [TestCase(false)]
+        public void Subscribe_WithInclusions(bool openAsync)
+        {
+            SyncTestHelpers.RunRosTestAsync(async () =>
+            {
+                using (var realm = await GetQueryBasedRealm(openAsync))
+                {
+                    var subscription = realm.All<ObjectB>()
+                                            .Where(b => b.BoolValue)
+                                            .Subscribe(includedBacklinks: b => b.As);
```

Comment:
I'm not familiar with this query syntax, but it looks good to me. How does it look if you wanted to included more than one relationship eg: b.As and b.Cs (if that relationship were named)?

Reply:
It would look like `.Subscribe(options, b => b.As, b => b.Cs)`. If you don't specify options, it's a bit more cumbersome: `.Subscribe(includedBacklinks: new[] { b => b.As, b => b.Cs })`.